package.json

```diff
@@ -12,12 +12,12 @@
     "start-evm": "ganache-cli -d -l 15000000 --allowUnlimitedContractSize --defaultBalanceEther 1000000 --mnemonic 'myth like bonus scare over problem client lizard pioneer submit female collect'",
     "submodule": "git submodule init && git submodule update",
     "lint": "eslint --ext ts .",
-    "test": "NODE_ENV=test && mocha -r ts-node/register 'test/src/**/*.test.ts' --exit && npm run test-mocked",
+    "test": "export NODE_ENV=test; mocha -r ts-node/register 'test/src/**/*.test.ts' --exit && npm run test-mocked",
```

Comment:
Did you find these exports were necessary in the end?

Reply:
I investigated further, it seems that "NODE_ENV=test <command>" works, but not "NODE_ENV=test && command"; with export both versions work, which is why I was getting more consistent behavior, as we had a mix.
I removed the export and the also the extra &&.